extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -471,13 +476,35 @@ export class AmpStoryPage extends AMP.BaseElement {
   }
 
   /**
-   * Finds embedded components in page and prepares them for their expanded view
-   * animation.
+   * Finds embedded components in page and prepares them.
    * @param {boolean=} forceResize
    * @private
    */
   findAndPrepareEmbeddedComponents_(forceResize = false) {
-    scopedQuerySelectorAll(this.element, EMBEDDED_COMPONENTS_SELECTORS)
+    this.addClickShieldToEmbeds_();
+    this.resizeInteractiveEmbeds_(forceResize);
+  }
+
+  /**
+   * Adds a pseudo element on top of the embed to block clicks from going into
+   * the iframe.
+   */
+  addClickShieldToEmbeds_() {
+    scopedQuerySelectorAll(this.element, EMBEDDED_COMPONENTS_SELECTORS).forEach(
+        el => {
+          this.resources_.mutateElement(el, () => {
```

Comment:
Calling `mutateElement` has a bit of overhead in the runtime. Could we query the elements, and then call mutateElement only once?